src/app/functions/server/dashboard.ts

```diff
@@ -311,11 +430,8 @@ function sanitizeExposures(
   return sanitizedExposures;
 }
 
-export function getExposureReduction(
-  summary: DashboardSummary,
-  scanResultItems: OnerepScanResultRow[]
-): number {
-  const countOfDataBrokerProfiles = scanResultItems.length;
-  const totalExposures = summary.totalExposures;
-  return Math.round((countOfDataBrokerProfiles / totalExposures) * 100);
+export function getExposureReduction(summary: DashboardSummary): number {
+  return Math.round(
+    (summary.dataBrokerTotalExposuresNum / summary.totalExposures) * 100
```

Comment:
Q: Any chance `totalExposures` can be zero here and cause a rift in the universe?

Reply:
Thanks Peter, added edge case and test